re/lib/dune

```diff
@@ -0,0 +1,6 @@
+(library
+  (name EsyBashLib)
+  (flags (-w -40 -w +26 -cclib -lunix))
+  (ocamlc_flags -custom unix.cma)
```

Comment:
Why do we need this? We are not going to compile it to bytecode, do we?

Reply:
`-cclib -lunix` are needed for `Unix`.

https://caml.inria.fr/pub/docs/oreilly-book/html/book-ora165.html

Is something unnecessary? New to linking non standard libs. Maybe I'm doing something unnecessary.